frontend/uddhar-frontend/src/coordinator/components/OngoingEvent.jsx

```
@@ -1,33 +1,50 @@
   return (
     <div
-      className={`min-w-70 h-50 ${bg} flex flex-col gap-y-[10px] text-left p-10 rounded cursor-pointer`}
+      className={`min-w-70 h-50 ${
+        currentEvent.disaster_id === info.disaster_id ? "bg-green-400" : bg
+      } flex flex-col gap-y-[10px] text-left p-10 rounded cursor-pointer`}
       onClick={() => {
-        onClickEventHandler(info.location);
+        onClickEventHandler(info);
       }}
       role="button"
     >
```

Biome (1.9.4)

[error] The HTML element with the interactive role "button" is not focusable.

A non-interactive HTML element that is not focusable may not be reachable for users that rely on keyboard navigation, even with an added role like "button".
Add a tabIndex attribute to make this element focusable.

(lint/a11y/useFocusableInteractive)